app/views/dashboard.html

```diff
@@ -108,10 +108,10 @@ <h4 translate="legend.ptratio">Pupil-Teacher Ratio</h4>
     <div class="row map-widget-row">
       <div class="col-sm-12 map-container">
         <ul class="nav nav-tabs" id="nav-tabs-dashboard">
-          <li role="presentation" ng-class="{active: polyType==='regions'}">
-            <a ng-click="togglePolygons('regions')">
-              <span class="hide-sm" translate="tab.regional-view">Regions</span>
-              <span class="show-sm" translate="tab.regional">Regional</span>
+          <li role="presentation" ng-class="{active: viewMode === 'schools'}">
+            <a ng-click="setViewMode('schools');polyType = null;">
```

Comment:
Is the `polytype = null` necessary? If so, that statement should go in `setViewMode` function, not in the view.

Could probably use a comment (even just a TODO) about it too, seems odd.